lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -235,6 +235,13 @@ def bash_systemd_polling_functions
         return command
       end
 
+      def get_ssh_group(ssh_user)
```

Comment:
It's feasible, but pretty much everywhere in this code base assumes ssh_group == ssh_user; actually determining it adds another RTT of latency and more scope for a network glitch.  It's tradition on Linux systems that users have an identically named group; it's tradition on Windows/Cygwin that Administrator's group is Administrators.